### ISSUE TITLE
Is it possible to show sub-mat prices for things like Tailoring bolts?

### ISSUE THREAD

**Issue by rushtothesun** [b-morgan/Skillet-Classic#156]
![wow](https://github.com/b-morgan/Skillet-Classic/assets/20650932/01b78456-db89-41ea-b90a-33a497eb744a)

For example in this picture Skillet is pulling the auction data from Auctionator for Bolt of Mageweave rather than the x25 Mageweave Cloth that's needed to make x5 Bolt of Mageweave. Is this something that can be shown?

**Comment by b-morgan** [comment]
Currently, the only option is to click on the "Bolt of Mageweave" which will then display the cost of 1 bolt as the buyout price and the cost of the cloth to make it. There will be a back button to return to the original recipe.

The biggest problem with having Skillet-Classic do this recursion automatically is that the frame where the results are displayed is limited in size and doesn't have scroll bars. Some professions (or some recipes) might fit but Engineering, for example, will overflow the space on a regular basis.

I have attempted to change that frame to one with scroll bars but I've never been successful. I also attempted to do the recursion but I never got that working either (Engineering got too complicated).

I'll take a look at both again but don't expect anything soon.
